whitepaper/whitepaper.md

```diff
@@ -371,14 +374,63 @@ The following table summarizes the use cases that will be explained in more deta
+| Use Case Title             | Car Trade |
+| Use Case                   | TBD |
+| Interworking patterns      | TBD |
+| Type of Social Interaction | TBD |
+| Narrative                  | TBD |
+| Actors                     | TBD |
+| Goals of Actors            | TBD |
+| Success Scenario           | TBD |
+| Success Criteria           | TBD |
+| Failure Criteria           | TBD |
+| Prerequisites              | TBD |
+| Comments                   |     |
+
+## 3.2 Electricity Trade
 
-## 3.1 Ethereum to Quorum Asset Transfer
+| Use Case Attribute Name    | Use Case Attribute Value                                                                                                                                                                                                                                                                                                                                                                                                                         |
+| -------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Use Case Title             | Electricity Trade |
+| Use Case                   | TBD |
+| Interworking patterns      | TBD |
+| Type of Social Interaction | TBD |
+| Narrative                  | TBD |
+| Actors                     | TBD |
+| Goals of Actors            | TBD |
+| Success Scenario           | TBD |
+| Success Criteria           | TBD |
+| Failure Criteria           | TBD |
+| Prerequisites              | TBD |
+| Comments                   |     |
+
+## 3.3 Supply chain
+
+| Use Case Attribute Name    | Use Case Attribute Value                                                                                                                                                                                                                                                                                                                                                                                                                         |
+| -------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Use Case Title             | Supply Chain |
```

Comment:
@takeutak Did you want me to fill this one out or are you still working on it?

Reply:
@petermetz I made a section template first In this PR. I think filling the contents of the table should be done by another PR.

Reply:
@takeutak Understood, thank you for clarifying!